docs/integration/solr.md

```diff
@@ -1,8 +1,114 @@
 title: Solr Monitoring Integration
 description:  Our monitoring and logging platform includes integration for Solr. Use predefined key metrics reports combined with rich data visualization tools to monitor critical Solr issues, and receive alerts on memory usage, uptime, load averages, index stats, document and filter caches, latency, rate, and more
 
+Sematext offers simple and versatile Solr monitoring agent written in Java and Golang without CPU and memory overhead on your application. It's easy to install and require no changes in Solr source code or your application source code. 
+
+## Sematext Solr Monitoring Agent 
+This lightweight, open-source [Java Monitoring Agent](https://github.com/sematext/sematext-agent-java) collects Solr performance metrics and sends them to Sematext. The second part of the Sematext Solr Monitoring Agent is a Golang-based agent responsible for Operating System level metrics like network, I/O and more. 
```

Comment:
Maybe phrase this a bit differently?

> The second part...

Like: "It comes packacaged with Golang-based agent (link to STA docs) responsible..."

And also mention it can be installed either with rpm/deb package on any linux machine or in container environments using sematext/spm-client etc etc...?

Reply:
Good point, will do.

Reply:
Done.